plio/spatial/transformations.py

```diff
@@ -171,9 +171,10 @@ def get_axis(file):
 
         return eRadius, pRadius
 
-def lat_ISIS_coord(record, semi_major, semi_minor):
+def reproject(record, semi_major, semi_minor, source_proj, dest_proj, **kwargs):
     """
-    Function to convert lat_Y_North to ISIS_lat
+    Transforms a set of coordinates (y, x, z) from one coordinate system to a different
+    coordinate system. Also handles ocentric to ographic transformations, and vise versa.
```

Comment:
We need a little more info here on how this handles ocentric/ographic because that handling is under-the-hood. @dpmayerUSGS might be able to offer some additional text/insight into how this is being handled.

Reply:
@acpaquette Can you update the PR.  Then let's merge this.  I know coverage will drop, but we can then merge #57 to start pushing coverage up.